src/main/resources/application-integration.properties

```diff
@@ -0,0 +1,42 @@
+transcription.api.url=http://fresenia.man.poznan.pl/tp-api
```

Comment:
I am not sure whether in dev props there are correct URLs but we should use dev tp-api, not the production one

Reply:
changed